### ISSUE TITLE
DNU after cloning a repo and switching to the Packages tab

### ISSUE THREAD

**Issue by gcotelli** [pharo-vcs/iceberg#369]
*IceLocalBranch(Object)>>doesNotUnderstand: #entity*

I've just cloned git@github.com:ba-st/Buoy.git in the standard location using "source" as the code subdirectory.

Find attached the stack fueled out.

[stack.tar.gz](https://github.com/pharo-vcs/iceberg/files/1059391/stack.tar.gz)

Image
-----
/home/gcotelli/Pharo/Pharo6-64/Pharo.image
Pharo6.0
Latest update: #60499
Unnamed

Virtual Machine
---------------
/home/gcotelli/Pharo/Pharo6-64/pharo-vm/lib/pharo/5.0-201705310241/pharo
CoInterpreter VMMaker.oscog-eem.2231 uuid: de62947a-7f40-4977-a232-e06a3a80c939 May 31 2017
StackToRegisterMappingCogit VMMaker.oscog-eem.2227 uuid: 7ea146b4-39ce-4de7-afa3-a76ed1d1da35 May 31 2017
VM: 201705310241 https://github.com/OpenSmalltalk/opensmalltalk-vm.git $ Date: Tue May 30 19:41:27 2017 -0700 $ Plugins: 201705310241 https://github.com/OpenSmalltalk/opensmalltalk-vm.git $

Unix built on May 31 2017 04:56:05 Compiler: 4.6.3
VMMaker versionString VM: 201705310241 https://github.com/OpenSmalltalk/opensmalltalk-vm.git $ Date: Tue May 30 19:41:27 2017 -0700 $ Plugins: 201705310241 https://github.com/OpenSmalltalk/opensmalltalk-vm.git $
CoInterpreter VMMaker.oscog-eem.2231 uuid: de62947a-7f40-4977-a232-e06a3a80c939 May 31 2017
StackToRegisterMappingCogit VMMaker.oscog-eem.2227 uuid: 7ea146b4-39ce-4de7-afa3-a76ed1d1da35 May 31 2017

**Comment by gcotelli** [comment]
I have to note that this repository is an old-fashioned filetree repository saved without using Iceberg.

**Comment by estebanlm** [comment]
yeah, this is a glamour problem, not iceberg :)
(Something happens when you change tabs and the entity is not updated properly)

It has nothing to do with a problem on iceberg :P

**Comment by tesonep** [comment]
I have checked, and the problem is sending the selection in the other tab.
Clearly is a problem of Glamour.

I also think it is a known problem, but it is not fixed. I have found
similar code only running is the presentation is of the correct class.

On 9 Jun 2017 07:23, "Esteban Lorenzano" <notifications@github.com> wrote:

yeah, this is a glamour problem, not iceberg :)
(Something happens when you change tabs and the entity is not updated
properly)

It has nothing to do with a problem on iceberg :P

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/pharo-vcs/iceberg/issues/369#issuecomment-307298253>,
or mute the thread
